fre/catalog/frecatalog.py

```diff
@@ -25,17 +30,20 @@ def catalogCli():
 @click.pass_context
 def builder(context, input_path = None, output_path = None, config = None, filter_realm = None,
             filter_freq = None, filter_chunk = None, overwrite = False, append = False):
+    # pylint: disable=unused-argument
     """ - Generate .csv and .json files for catalog """
-    context.forward(gen_intake_gfdl.main)
+    context.forward(gen_intake_gfdl.create_catalog_cli)
```

Comment:
this is the update!

Reply:
and and and and 

https://github.com/NOAA-GFDL/fre-cli/blob/d237b690cdd4e593cbdea2202ea2ec7578baba47/fre/tests/test_fre_catalog_cli.py